src/modules/remote.json

```diff
@@ -0,0 +1,151 @@
+{
+    "openrpc": "1.2.4",
+    "info": {
+      "title": "Remote",
+      "description": "A module for managing Remote keys, paring and programming.",
+      "version": "0.0.0"
+    },
+    "methods": [
+      {
+        "name": "state",
+        "summary": "Read-only property method for the remote control state. This is a getter for the state or a way to subscribe to changes to the state.",
+        "params": [],
+        "result": {
+          "name": "remoteList",
+          "summary": "Contains a list of remotes controls paired to the device.",
+          "schema": {
+            "$ref": "#/components/schemas/RemoteControlState"
+          }
+        },
+        "examples": [
+          {
+            "name": "Remote State Example",
+            "params": [],
+            "result": {
+              "remoteList": [
+                {
+                  "model": "PR2",
+                  "batteryPercentage": "70"
+                }
+              ]
+            }
+          }
+        ]
+      },
+      {
+        "name": "startPairing",
+        "summary": "Setup the device to allow Remote control to be paired on the next keypress.",
+        "params": [],
+        "result": {
+          "name": "remotePairState",
+          "summary": "The state of currently paired remote",
+          "schema": {
+            "$ref": "#/components/schemas/RemotePairState"
+          }
+        },
+        "examples": [
+          {
+            "name": "Remote State Examples",
+            "params": [],
+            "result": {
+              "pairingState": "pairingRemote"
+            }
+          }
+        ]
+      },
+      {
+        "name": "onRemotePairing",
+        "tags": [
+          {
+            "name": "event"
+          }
+        ],
+        "summary": "Listen for events when the remote control is pairing.",
+        "params": [
+          {
+            "name": "listen",
+            "required": true,
+            "summary": "whether to listen, or stop listening",
+            "schema": {
+              "type": "boolean"
+            }
+          }
+        ],
+        "result": {
+          "name": "remotePairState",
+          "summary": "The state of currently paired remote",
+          "schema": {
+            "$ref": "#/components/schemas/RemotePairState"
```

Comment:
There is a ListenResponse Object in the `firebolt-schemas` Types document.

It is the first response of every event API, and it tells the client that the registration was successful.

We use a `oneOf` array in the `result.schema` to include the ListenerResponse as well as the real event result for all listeners to ensure that the first success response is consistent.

```
"schema": {
  "oneOf": [
    {
      "$ref": "https://meta.comcast.com/firebolt/types#/definitions/ListenResponse"
    },
    {
      "$ref": "#/components/schemas/RemotePairState"
    }
  ]
} 
```